### PR TITLE
Add gnufied to ebs CSI driver

### PR DESCRIPTION
**Pull request by gnufied**
Adding myself to ebs csi driver maintainer list.

### CODE REVIEW

File: config/kubernetes-sigs/sig-aws/teams.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ teams:
     - d-nishi
     - jsafrane
     - leakingtapan
+    - gnufied
```
Comment (nikhita) [inline config/kubernetes-sigs/sig-aws/teams.yaml:15]: This needs to be alphabetically sorted and you'll also need to add yourself as a @kubernetes-sigs member here - https://github.com/kubernetes/org/blob/master/config/kubernetes-sigs/org.yaml

**Comment by nikhita** [comment]
Super tiny nit (and I apologize for the back and forth!) -- could you split the org.yaml change into a separate commit with the commit message `Add gnufied to kubernetes-sigs` and keep the teams change in a separate commit? We've been maintaining commit history in that manner and I want us to remain consistent with that practice :)

**Comment by gnufied** [comment]
@nikhita no worries and done.

**Comment by nikhita** [comment]
/approve

Leaving lgtm to @leakingtapan and @jsafrane for the ebs csi driver team update.
/assign @leakingtapan @jsafrane

**Pull request merged**